activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -171,6 +173,10 @@ def supports_explain?
         true
       end
 
+      def supports_foreign_keys?
+        true
+      end
+
```

Comment:
Put a version test here so you're not returning `true` erroneously for older versions that don't support it.